### PR TITLE
FXIOS-1381 ⁃ FXIOS-1166: closes #7605 Confirmation snackbars are missing from address bar context menu action: "Copy Address".

### LINKED ISSUE #7605: FXIOS-1166 ⁃ [Bug] Confirmation snackbars are missing from address bar context menu action: "Copy Address"
## Steps to reproduce

1. “Force touch” or long-press the address bar while a website is currently open
2. Select "Copy Address"

### Expected behavior

If you select "Copy Address" from the Page Actions Menu (⋯), it will show a confirmation snackbar. The same selection, when done from the address bar, should also show a confirmation snackbar.

### Actual behavior

Nothing is displayed

### Device & build information
* Device: all

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1166)

### PR DESCRIPTION
**Pull request by chriscoderdr**
Closes #7605 
Added toast confirmation when copy address as per requirement.

https://user-images.githubusercontent.com/2073238/103979954-e4079a80-514c-11eb-9a7a-0287965dc8ea.mov

Demo Video

┆Issue is synchronized with this [Jira Task](https://jira.mozilla.com/browse/FXIOS-1381)

### CODE REVIEW

File: Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift [hunk 1]
```diff
@@ -63,6 +63,9 @@ extension PhotonActionSheetProtocol {
         let copyAddressAction = PhotonActionSheetItem(title: Strings.CopyAddressTitle, iconString: "menu-Copy-Link") { _, _ in
             if let url = self.tabManager.selectedTab?.canonicalURL?.displayURL ?? urlBar.currentURL {
                 UIPasteboard.general.url = url
+                if let view = urlBar.window {
```
Comment (dnarcese) [inline Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift:66]: What is the reason for this check?

Comment (chriscoderdr) [reply Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift:66]: urlBar.window is type UIWindow? (Optional)

Comment (chriscoderdr) [reply Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift:66]: image

Comment (dnarcese) [reply Client/Frontend/Widgets/PhotonActionSheet/PhotonActionSheetProtocol.swift:66]: Oh sorry, yes I see, thanks

**Review by dnarcese — APPROVED** [review]

**Review by dnarcese — CHANGES_REQUESTED** [review]
Can we get it to anchor on top of the urlbar instead? This is how the other snackbars behave
image

**Comment by chriscoderdr** [comment]
> Can we get it to anchor on top of the bottom toolbar instead? This is how the other snackbars behave
> image

Sure, didn't notice, will make the change.

**Comment by chriscoderdr** [comment]
> Can we get it to anchor on top of the bottom toolbar instead? This is how the other snackbars behave
> image

simulator_screenshot_E39A7FC9-4F67-4431-A168-213895911242

Fixed

https://user-images.githubusercontent.com/2073238/104053139-be17df80-51b8-11eb-9971-885d790a6dc8.mov

**Review by dnarcese — APPROVED** [review]

**Pull request merged**